app/views/tasks/show.html.slim

```diff
@@ -29,7 +29,7 @@
               - else
                 - [*1..5].each do |i|
                   span.fa.fa-star-o data-rating=i
-    = render 'remove_state_tag', exercise: @task
+    /= render 'remove_state_tag', exercise: @task
```

Comment:
What exactly was this `remove_state_tag` for? I cannot find it in the code any longer...